source/Personality.h

```diff
@@ -41,7 +41,8 @@ class Personality {
 	bool IsPacifist() const;
 	bool IsForbearing() const;
 	bool IsTimid() const;
-	bool IsHeroic() const;
+	bool IsHunting() const;
+	bool IsDaring() const;
 	bool IsNemesis() const;
```

Comment:
personalities functions should be in the same order as in the TOKEN map.
```suggestion
	bool IsNemesis() const;
	bool IsDaring() const;
```